src/include/rados/rgw_file.h

```diff
@@ -27,7 +27,7 @@ extern "C" {
 
 #define LIBRGW_FILE_VER_MAJOR 1
 #define LIBRGW_FILE_VER_MINOR 1
-#define LIBRGW_FILE_VER_EXTRA 4
+#define LIBRGW_FILE_VER_EXTRA 6
```

Comment:
this and the other pr both try to change the same header, can we you include https://github.com/ceph/ceph/pull/18800 also in this pr itself to avoid merge conflict